### PR TITLE
(refactor) Update todo example to Polymer 2.

### PR DESCRIPTION
**Pull request by hastebrot**
**Work progress:** 

- [x] The `bower_components` directory needs to be updated using a clean `bower install`. 
- [x] The `polymer-2` branch in the `dolphin-platform-polymer` repository needs bugfixes to be merged.

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/664)

### CODE REVIEW

File: platform-examples/todo-example/polymer-client/index.html [hunk 1]
```diff
@@ -1,30 +1,47 @@
...
+    body {
+      font-family: "Roboto", "Noto", sans-serif;
+      font-size: 14px;
+      margin: 0;
+      padding: 24px;
+    }
+  </style>
+
+  <script>
+    function createClientContext(url) {
```
Comment (hendrikebbers) [inline platform-examples/todo-example/polymer-client/index.html]: Is there a good way to provide this as a static / global method in dolphin-platform-js ?

File: platform-examples/todo-example/polymer-client/todo-view.html [hunk 2]
```diff
@@ -1,74 +1,83 @@
...
-<head>
-    <link rel="import" href="bower_components/paper-input/paper-input.html">
-    <link rel="import" href="bower_components/paper-button/paper-button.html">
-    <link rel="import" href="bower_components/paper-item/paper-item.html">
-    <link rel="import" href="bower_components/paper-item/paper-item-body.html">
-    <link rel="import" href="bower_components/iron-icon/iron-icon.html">
-    <link rel="import" href="bower_components/iron-icons/iron-icons.html">
-    <link rel="import" href="bower_components/paper-icon-button/paper-icon-button.html">
-</head>
+<link rel="import" href="bower_components/polymer/polymer.html">
```
Comment (hendrikebbers) [inline platform-examples/todo-example/polymer-client/todo-view.html]: Do we really need this? Normally this should come as a transitive dependency or not?

File: platform-examples/todo-example/todo-server-spring/src/main/resources/application.properties [hunk 3]
```diff
@@ -21,3 +21,4 @@ logging.level.com.canoo.dolphin.server.context=trace
 logging.level.com.canoo.dp.impl.server.config.DefaultPlatformConfiguration=debug
 
 dolphinPlatform.showRemotingLogging=false
+dolphinPlatform.maxClientsPerSession=9999
```
Comment (hendrikebbers) [inline platform-examples/todo-example/todo-server-spring/src/main/resources/application.properties]: should be removed before merge

File: platform-examples/todo-example/polymer-client/todo-view.html (hunk 2 above)
Comment (hastebrot) [reply platform-examples/todo-example/polymer-client/todo-view.html]: Indeed, we do not need `polymer.html` to be imported. 

However I propose to keep it per convention and for consistency. The elements in the Polymer shop example also do this (https://github.com/Polymer/shop/tree/master/src).

I changed the import from `polymer.html` to `polymer-element.html` to emphasize we are defining a Polymer 2 element.

File: platform-examples/todo-example/polymer-client/index.html (hunk 1 above)
Comment (hastebrot) [reply platform-examples/todo-example/polymer-client/index.html]: We could add an `<dolphin-client>` elements that contains a `clientContext` property and also add this property to global `window` (a bit ugly, through).

**Comment by hastebrot** [comment]
@hendrikebbers The latest commits change the `font-roboto` dependency to `font-roboto-local` to prevent requests to fonts.googleapis.com being send in the examples.

**Pull request merged**